torch/distributions/log_normal.py

```diff
@@ -54,7 +54,7 @@ def mode(self):
 
     @property
     def variance(self):
-        return (self.scale.pow(2).exp() - 1) * (2 * self.loc + self.scale.pow(2)).exp()
+        return self.scale.pow(2).expm1() * (2 * self.loc + self.scale.pow(2)).exp()
```

Comment:
Now that you're at it, could you factor out that `self.scale.pow(2)` to avoid computing it twice?